### PR TITLE
[Outlook](NAA) Add section about identity and your services

### PR DESCRIPTION
**Pull request by davidchesnut**
Documents a common question about moving from the old Exchange token scenario of validating the ID token to use it in services.

### CODE REVIEW

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ If you submit an issue, please include the following information.
 - Version of msal-browser.
 - Logs from msal-browser.
 
-## Developer troubleshooting questions
+## Developer questions
```
Comment (lindalu-MSFT) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md:168]:
```suggestion

```
I don't think you need this heading level.

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ If you submit an issue, please include the following information.
...
 
 ### How do I get more debug information from MSAL and NAA?
```
Comment (lindalu-MSFT) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md:170]:
```suggestion
## How do I get more debug information from MSAL and NAA?
```

**Review by lindalu-MSFT — COMMENTED** [review]
A few suggestions to commit.

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md (hunk 1 above)
Comment (davidchesnut) [reply docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md:168]: It adds it to the top inline TOC which is handy to keep this separate from the admin section.

**Pull request merged**